lib/mackerel/metric.rb

```diff
@@ -0,0 +1,80 @@
+module Mackerel
+  class Metric
```

Comment:
Is this class used?
(And this class does not represent metric structures used by Mackerel)

Reply:
No. I deleted the class.